docs/recipes/contribute-recipes.md

```diff
@@ -0,0 +1,75 @@
+---
+description: Share your Prefect solution recipes with the Prefect Community.
+tags:
+    - contributing
+    - recipes
+    - examples
+    - use cases
+---
+
+# Share Your Solutions with the Prefect Community
+
+[Prefect recipes](https://github.com/PrefectHQ/prefect-recipes) provide a vital cookbook from which Prefect users can find helpful code examples and, when appropriate, common steps to Prefect solutions for specific use cases. 
+
+We love submissions for recipes from anyone who has an example that another Prefect user can follow to achieve a common Prefect solution.
+
+## What is a recipe?
+
+A Prefect recipe is like a cookbook recipe: it tells you what you need &mdash; the ingredients &mdash; and some basic steps, but assumes you can put the pieces together. Think of the Hello Fresh meal experience, but for dataflows.
+
+A tutorial, on the other hand, is Julia Child holding your hand through the entire cooking process: explaining each ingredient and procedure, demonstrating best practices, pointing out potential problems, and generally making sure you can’t stray from the happy path to a delicious meal.
+
+We love Julia, and we love tutorials. But we don’t expect that a Prefect recipe should handhold users through every step and possible contingency of a solution. A recipe can start from an expectation of more expertise and problem-solving ability on the part of the reader.
+
+To see an example of a high quality recipe, check out **[Serverless with AWS Chalice](https://github.com/PrefectHQ/prefect-recipes/tree/main/flows-advanced/serverless)**. This recipe includes all of the elements we like to see.
+
+## What to do, explained in a brief recipe
```

Comment:
I don't want to give users 2 definitions of recipes, because we just told them what a recipe was and then we go on to say it means something different in this heading. 

```suggestion
## Steps to adding your recipe
```